src/app/@ansyn/ol/maps/helpers/helpers.ts

```diff
@@ -0,0 +1,43 @@
+import { Observable } from 'rxjs';
+
+export function exportMapHelper(mapObject, size, resolution, classToInclude): Observable<HTMLCanvasElement> {
+	const width = Math.round((size[0] * resolution) / 25.4);
```

Comment:
can you elaborate on this function?
and what does 'helper' mean in the context of the project

Reply:
it contain function that the help the ol module.
export map, is the same for `ol-map` and `disable-ol-map`